compact/ecal_forward_homogeneous.xml

```diff
@@ -0,0 +1,92 @@
+<lccdd>
+
+  <info name="ecal_forward_homogeneous.xml"
+        title="Homogeneous forward ECal"
+        author="@rymilton"
+        url="https://github.com/rymilton"
```

Comment:
This should be changed into your username and github account.

Reply:
I changed to my GitHub account.